Cargo.toml

```diff
@@ -17,7 +17,7 @@ name = "dbt_extractor"
 crate-type = ["cdylib", "lib"]
 
 [dependencies]
-pyo3 = { version = "0.15.1", features = ["abi3-py36", "extension-module"] }
```

Comment:
3.7 EOL is at the end of the month so we can skip 3.7 too.

Reply:
*looks at calendar* How is it June already. Fixed.